src/core/BigInt.c

```
@@ -1,759 +1,759 @@
+	for(i = 0; X.m_ulValue[X.m_nLength-1]>0; i ++)
+	{
+		a=Mod_Big_Long(X,system);
+		s[i] = t[a]; 
+		//Mov_Big_Big(&X,Div_Big_Long(X,system));
+		Div_Big_Long(&X,X,system);
+	}
+	len = i-1;  
+	for( i= 0; i<= len; i++)
+		s1[i] = s[len-i];
+	str = s1;
+	return str;
```

Cppcheck (2.20.0)

[error] Returning pointer to local variable 's1' that will be invalid when returning.

(returnDanglingLifetime)